src/main/java/com/bookstore/bookstoreapi/data/entity/Rating.java

```diff
@@ -0,0 +1,41 @@
+package com.bookstore.bookstoreapi.data.entity;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Entity
+@Table(name = "Rating")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Rating {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "RatingID")
+    private Long id;
+
+    @Column(name = "Book")

```

Comment:
By our code standards, relationships belong in the bottom, and entity attributes belong at the top, so when you switch this to a relationship with the book entity, please move this code to the bottom.